muonDIS/makeMuonDIS_AM.py

```
@@ -0,0 +1,387 @@
+
+    try:
+        muon_tree = muonFile.MuonAndSoftInteractions
+    except Exception as e:
+        logging.error(e)
+        muonFile.Close()
+        exit(1)
+
+    logging.debug(f"Total entries in the tree: {muon_tree.GetEntries()}")
+    last_mu_event = min(muon_tree.GetEntries(), first_mu_event + n_events)
+
+    logging.info("Creating output file: ", args.outputFile)
```

Ruff (0.15.20)

[error] Too many arguments for `logging` format string

(PLE1205)